roboflow/core/version.py

```diff
@@ -158,10 +159,15 @@ def download(self, model_format=None, location=None, overwrite: bool = True):
         self.__wait_if_generating()
 
         if model_format == "yolov8":
-            # we assume the user will want to use yolov8, for now we only support ultralytics=="8.11.0"
-            print_warn_for_wrong_dependencies_versions(
-                [("ultralytics", "<=", "8.0.20")]
-            )
+            # if ultralytics is installed, we will assume users will want to use yolov8 and we check for the supported version
+            try:
+                import_module("ultralytics")
+                print_warn_for_wrong_dependencies_versions(
+                    [("ultralytics", "<=", "8.0.20")]
+                )
+            except ImportError as e:
+                # silently fail
+                pass
```

Comment:
We could keep it as it is. Or add some info about our ultralytics support.

Reply:
Happy to do both, since you have been exposed to the final user more than me, what do you think we should do?

Reply:
Copy that

Reply:
You will need to install it to call `.deploy` later, you can use: `pip install ultralytics<=8.0.20`.

to

Roboflow `.deploy` supports only models trained with `ultralytics<=8.0.20`

Thoughts?

Reply:
I've merged the two messages :angel: